Notebook deploy is broken

```
+ CHANGED=yes
+ [[ -e notebook/get-deployed-sha.sh ]]
+ [[ yes != no ]]
+ cd notebook
+ /bin/bash hail-ci-deploy.sh
cat: notebook-image: No such file or directory
sed -e "s,@sha@,17a365c57d0f," \
  -e "s,@image@,," \
  < deployment.yaml.in > deployment.yaml
kubectl apply -f deployment.yaml
Error from server (Forbidden): error when retrieving current configuration of:
Resource: "/v1, Resource=services", GroupVersionKind: "/v1, Kind=Service"
Name: "notebook", Namespace: "batch-pods"
Object: &{map["metadata":map["labels":map["app":"notebook"] "name":"notebook" "namespace":"batch-pods" "annotations":map["kubectl.kubernetes.io/last-applied-configuration":""]] "spec":map["ports":[map["port":'P' "protocol":"TCP" "targetPort":'\u1388']] "selector":map["app":"notebook"]] "apiVersion":"v1" "kind":"Service"]}
from server for: "deployment.yaml": services "notebook" is forbidden: User "system:serviceaccount:batch-pods:deploy-svc" cannot get services in the namespace "batch-pods": Unknown user "system:serviceaccount:batch-pods:deploy-svc"
Makefile:4: recipe for target 'deploy' failed
make: *** [deploy] Error 1
```

--- a/notebook/Makefile
+++ b/notebook/Makefile
@@ -4,4 +4,4 @@ deploy:
 	sed -e "s,@sha@,$(shell git rev-parse --short=12 HEAD)," \
 	  -e "s,@image@,$(shell cat notebook-image)," \
 	  < deployment.yaml.in > deployment.yaml
-	kubectl apply -f deployment.yaml -n default
+	kubectl -n default apply -f deployment.yaml

--- a/notebook/Makefile
+++ b/notebook/Makefile
@@ -4,4 +4,4 @@ deploy:
 	sed -e "s,@sha@,$(shell git rev-parse --short=12 HEAD)," \
 	  -e "s,@image@,$(shell cat notebook-image)," \
 	  < deployment.yaml.in > deployment.yaml
-	kubectl apply -f deployment.yaml
+	kubectl apply -f deployment.yaml -n default

